docs/sdk/plugins/JETI.md

```diff
@@ -0,0 +1,80 @@
+---
+title: JETI Extension
+sidebar_label: JETI Extension
+description: Use IPFS for Tableland data
+---
+
+## JavaScript Extension for Tableland and IPFS
+
+Using JETI with Tableland
+JETI (JavaScript Extension for Tableland and IPFS) allows you to easily add data to IPFS and read it in your applications while interacting with the Tableland network. The following guide will walk you through the process of integrating JETI into your project using JavaScript.
+
+### 1. Installation
+
+First, install the JETI package using npm or yarn:
+
+```
+npm i @tableland/jeti @tableland/sdk
+```
+
+### 2. Import the necessary modules from @tableland/sdk and @tableland/jeti:
```

Comment:
can probs just state `Import the necessary modules` and drop `from @tableland/sdk and @tableland/jeti:`